examples/httpserver_simpletest.py

```diff
@@ -21,16 +21,17 @@
 print("Connected to", ssid)
 
 pool = socketpool.SocketPool(wifi.radio)
-server = HTTPServer(pool, "/static")
+server = HTTPServer(pool, "")
```

Comment:
I think we should keep the `"/static"` value for the root_path parameter.

Setting it to empty means that files in the root of the CIRCUITPY drive will be served which is potential security concern since settings.toml would get served. 

The library was just recently updated to add additional restrictions to prevent that, so best practice is definitely to use a specific directory for the root path being served. 

Even in cases where there is no static content used it can still be set to `"/static"`.

Reply:
Totally makes sense, thanks for the feedback.  I'll get that updated shortly.